### PR TITLE
Remove usages of deprecated methods

### PR DESCRIPTION
**Pull request by vkolomeyko**
- Fix typos
- Fix other Kotlin warnings

### CODE REVIEW

File: client/rpc/src/main/kotlin/net/corda/client/rpc/internal/RPCClientProxyHandler.kt [hunk 1]
```diff
@@ -164,7 +165,7 @@ class RPCClientProxyHandler(
                 TimeUnit.MILLISECONDS
         )
         sessionAndProducerPool.run {
-            it.session.createTemporaryQueue(clientAddress, clientAddress)
+            it.session.createTemporaryQueue(clientAddress, ActiveMQDefaultConfiguration.getDefaultRoutingType(), clientAddress)
```
Comment (josecoll) [inline client/rpc/src/main/kotlin/net/corda/client/rpc/internal/RPCClientProxyHandler.kt]: What is the impact of explicitly specifying this parameter?
(eg. does this deviate from the default behaviour)

Comment (vkolomeyko) [inline client/rpc/src/main/kotlin/net/corda/client/rpc/internal/RPCClientProxyHandler.kt]: Explicit parameter specification was used to eliminate use of a deprecated method.
This does not deviate from the default implementation, please see: 
```
org.apache.activemq.artemis.core.client.impl.ClientSessionImpl#createTemporaryQueue(org.apache.activemq.artemis.api.core.SimpleString, org.apache.activemq.artemis.api.core.SimpleString, org.apache.activemq.artemis.api.core.SimpleString)
```

Comment (josecoll) [inline client/rpc/src/main/kotlin/net/corda/client/rpc/internal/RPCClientProxyHandler.kt]: hmmmm .... I'm not sure if this is correct as
`public static final RoutingType DEFAULT_ROUTING_TYPE = RoutingType.MULTICAST;`

Comment (mnesbit) [inline client/rpc/src/main/kotlin/net/corda/client/rpc/internal/RPCClientProxyHandler.kt]: https://activemq.apache.org/artemis/docs/2.0.0/address-model.html
Looks like Multi-cast is more pub-sub.

Comment (vkolomeyko) [inline client/rpc/src/main/kotlin/net/corda/client/rpc/internal/RPCClientProxyHandler.kt]: Looks like ANYCAST is a name point-to-point communication as per:
https://activemq.apache.org/artemis/docs/2.0.0/address-model.html

Comment (josecoll) [inline client/rpc/src/main/kotlin/net/corda/client/rpc/internal/RPCClientProxyHandler.kt]: Yes - an overloaded (and misleading) definition of multi-cast ;-)

**Pull request merged**